package.json

```diff
@@ -83,10 +88,16 @@
 	},
 	"ava": {
 		"files": [
-			"test/*.js"
+			"test/*.ts"
+		],
+		"babel": false,
+		"compileEnhancements": false,
+		"extensions": [
+			"js",
```

Comment:
Aren’t all test files just ts now?

Reply:
It's been removed. Good catch.

Reply:
Yup. Not sure why you're seeing that. It's been removed.